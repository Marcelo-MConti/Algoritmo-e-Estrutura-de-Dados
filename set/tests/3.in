0
100 100
597 596 594 586 582 575 574 554 549 545 540 536 531 530 525 522 519 518 515 511 510 507 505 504 497 490 487 483 481 471 468 464 463 447 442 441 440 439 423 420 417 398 397 394 392 381 379 373 371 361 352 349 323 309 303 302 299 298 297 282 280 279 276 262 260 250 245 240 235 233 231 230 222 219 214 213 208 206 203 192 186 184 173 171 167 162 161 148 145 143 142 136 128 127 120 119 115 107 104 100
44 504 138 568 368 113 798 99 28 345 575 447 280 736 451 164 167 415 587 215 202 787 325 418 715 469 65 553 620 574 705 149 546 362 651 351 535 376 476 738 426 238 370 759 385 503 580 317 688 89 21 93 52 50 237 152 292 125 75 199 165 342 18 352 386 751 251 322 479 303 338 297 96 597 599 640 312 724 17 339 636 330 140 27 314 739 679 135 201 148 559 217 168 522 290 717 430 632 334 132
3
