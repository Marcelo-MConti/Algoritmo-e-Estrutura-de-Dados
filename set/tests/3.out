148, 167, 280, 297, 303, 352, 447, 504, 522, 574, 575, 597, 
